apps/api/src/app/topics-v2/usecases/upsert-topic/upsert-topic.command.ts

```diff
@@ -1,12 +1,14 @@
-import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import { IsNotEmpty, IsOptional, IsString, Length } from 'class-validator';
 import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
 
 export class UpsertTopicCommand extends EnvironmentWithUserCommand {
   @IsString()
   @IsNotEmpty()
+  @Length(1, 100)
   key: string;
 
   @IsString()
   @IsOptional()
+  @Length(0, 100)
```

Comment:
```suggestion
  @Length(1, 100)
```

Reply:
This one is actually optional, i've also updated the other DTO with a proper tag